Units/parser-cxx.r/bug1575055.cpp.d/expected.tags

```diff
@@ -1,3 +1,3 @@
 MyClass	input.cpp	/^	class MyClass { };$/;"	c	namespace:TheNamespace	file:
 TheNamespace	input.cpp	/^namespace TheNamespace {$/;"	n	file:
-variable	input.cpp	/^	int variable;$/;"	m	namespace:TheNamespace	file:
+variable	input.cpp	/^	int variable;$/;"	v	namespace:TheNamespace
```

Comment:
file: is gone. I don't understand the semantics of namespace. Is this a declaration or definition?

About kind, look at README in the same directory. I agree with you; I think this should be v.
If you have your own idea about this, could you update README?